spec/lib/task-graph-spec.js

```diff
@@ -516,6 +516,14 @@ describe('Task Graph', function () {
                     .that.equals('finished');
             });
         });
+
+        it('should create task with waitOn.anyOf', function(){
```

Comment:
Please add another test case to validate if the anyOf is used as label, the validation will surely detect the error.